extensions/docs/source/extension.rst

```diff
@@ -0,0 +1,23 @@
+Hyperledger Fabric Extensions
+=============================
+
+Hyperledger Fabric Extensions are a set of enhancements to Hyperledger Fabric.
+The aim of these enhancements are to allow for performance enhancements on a
```

Comment:
performance enhancements (such as what you are listing) and feature enhancements (such as DCAS)...